sonar_dave_plugins/sonar_dave_uuv/src/gazebo_ros_image_sonar.cpp

```diff
@@ -909,59 +966,70 @@ cv::Mat GazeboRosImageSonar::ConstructScanImage(cv::Mat& depth, cv::Mat& SNR)
 cv::Mat GazeboRosImageSonar::ConstructVisualScanImage(cv::Mat& raw_scan)
 {
   float fov = depthCamera->HFOV().Degree();
-  float mapped_range = float(raw_scan.rows);
+  float mapped_range = static_cast<float>(raw_scan.rows);
 
   cv::Scalar blue(15, 48, 102);
   cv::Scalar black(0, 0, 0);
   cv::Mat scan(raw_scan.rows, raw_scan.cols, CV_8UC3);
   scan.setTo(blue);
-  
+
   cv::Point center(scan.cols/2, scan.rows);
   cv::Size axes(scan.rows+3, scan.rows+3);
-  cv::ellipse(scan, center, axes, -90, -fov/2., fov/2., black, -1); //, int lineType=LINE_8, 0);
+  cv::ellipse(scan, center, axes, -90, -fov/2., fov/2., black, -1);
```

Comment:
Please leave in author's comment unless it is no longer applicable.  In this case, the comment suggests a modification to the action that is currently not chosen.

Reply:
(I believe the commented code would just set the parameters to their default values. You're right this is slightly more than a stylistic tweak.)